src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/AsyncWriteProxyEx.cs

```diff
@@ -43,7 +43,6 @@ public AsyncReplayTimeoutException(string message)
         {
         }
 
-#if SERIALIZATION
```

Comment:
I can't tell you how thrilled I am to get rid of this crap @heatonmatthew - so far, according to the CI system, the compiler and the test runtimes seem pretty happy with these changes so I don't have much to add on each individual change here.